src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1038,25 +1048,22 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 
 				if (e.dataTransfer) {
 					e.dataTransfer.effectAllowed = 'copyMove';
+					e.dataTransfer.setDragImage(tab, 0, 0); // top left corner of dragged tab set to cursor position
```

Comment:
Why do we need to `setDragImage`?

Reply:
This makes sure that the tab which is dragged is not obfuscating the drop target markers.